test/gtest/ucs/test_bitops.cc

```diff
@@ -90,6 +90,8 @@ UCS_TEST_F(test_bitops, ptr_ctz) {
     ASSERT_EQ(88, ucs_count_ptr_trailing_zero_bits(buffer, 160));
 }
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wframe-larger-than="
```

Comment:
to avoid all pragma in test/gtest, is there a way to not set the -Wframe-larger-than when compiling gtests?

Reply:
There are a few macros that I can change to make those tests use the heap instead of the stack, just like I did in the commit "TEST/GTEST: reduce stack frame size in INSTANTIATE_TEST_SUITE_P macro".
Those macros are "UCS_TEST_P_" and "UCS_TEST_P_", I'm trying to make the syntax work, it's challenging so I added the pragma to see if I can make the CI pass meanwhile.
The reason I want to reduce the stack allocation in the tests is that later on I will check the stack usage on run time alongside running the tests and if the stack usage of the tests are initially above the threshold then it'll be harder to add a rule for the CI to catch cases of large stack usage during run time.

Reply:
Outdated, done in other PRs